File: Models/Package.cs
using System;
using System.Text.Json;

namespace GTrack.Models
{
    public class Package : IdentificableItem
    {
        public string uid { get; set; }

        public string code { get; set; }

        public Measure measures { get; set; }

        public string description { get; set; }

        public string inboundStageUid { get; set; }

        public string outboundStageUid { get; set; }

        public string inboundTripUid { get; set; }

        public string outboundTripUid { get; set; }

        public DateTime creationDate { get; set; }

        public string type { get; set; }

        public string estimatedDestinationArea { get; set; }

        public Address estimatedDestinationAddress { get; set; }

        public override string ToString()
        {
            return JsonSerializer.Serialize<Package>(this);
        }
    }

    public class Measure
    {
        public double weight { get; set; }
        public double length { get; set; }
        public double height { get; set; }
        public double width { get; set; }
     

        public override string ToString()
        {
            return JsonSerializer.Serialize<Measure>(this);
        }
    }


}

File: ClientApp/src/app/modules/content-modules/warehouse/models/package.ts
import { Address } from "../../trips/models/models";

export interface Package {
    uid: string,
    code: string,
    measures: Measures,
    description: string,
    inboundStageUid: string | null,
    outboundStageUid: string | null,
    inboundTripUid: string | null,
    outboundTripUid: string | null,
    creationDate: Date,
    type: PackageType,
    estimatedDestinationArea: string,
    estimatedDestinationAddress : Address
}

export interface Measures{
    weight: number,
    length: number,
    height: number,
    width: number,
}

export enum PackageType {
    pedane = "Pedane",
    package = "Package"
}